src/resources/user.js

```diff
@@ -42,5 +42,19 @@ export default api => (
     static all() {
       return this.notImplemented('all');
     }
+
+    async updateBrand(param, id) {
```

Comment:
1. `param` needs to be plural - `params`
2. Why are we passing in an `id` parameter here? We should be getting the instance ID of the user.

Reply:
Corrected